### DISCUSSION TITLE
Recording Working Hours for Multiple Projects in a Single Day

### DISCUSSION THREAD

**Discussion by loura67508** [kimai/kimai#4900] [General]
Hello,
I enter the working hours of our employees on the desktop PC. Sometimes, an employee works 
on multiple projects in one day. Do I need to create a separate entry for each project on that day, 
or is it possible to record multiple projects and their respective working hours in a single entry?

**Comment by kevinpapst** [comment]
> create a separate entry for each project on that day

This 👍

**Comment by loura67508** [comment]
Ok, thanks! Can Kimai be set so that activity is not a mandatory field. (My activity is always the same...)

**Reply by kevinpapst** [reply]
No. 
But if you have only one global activity, that one should always be pre-selected.

**Comment by loura67508** [comment]
OK Thank you very much!
